app/views/candidate_interface/other_qualifications/review/show.html.erb

```diff
@@ -6,13 +6,13 @@
     <h1 class="govuk-heading-xl">
```

Comment:
Can’t make a suggestion for all this rows… this heading and button content shouldn’t be within a grid container, so lets remove that (i.e the heading can go the full width of the page).